<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>register_info</class>
 <widget class="QDialog" name="register_info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign up  page</string>
  </property>
  <property name="windowIcon">
   <iconset resource="game_recources.qrc">
    <normaloff>:/game_backgrounds/pics_project/pngwing.com.png</normaloff>:/game_backgrounds/pics_project/pngwing.com.png</iconset>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color:rgb(59, 59, 59);
border-top-left-radius:5px; 
border-bottom-left-radius:5px; 
color:rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string> Name </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color:rgb(59, 59, 59);
border-top-left-radius:5px; 
border-bottom-left-radius:5px; 
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string> Username </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color:rgb(59, 59, 59);
border-top-left-radius:5px; 
border-bottom-left-radius:5px; 
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string> Password </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color:rgb(59, 59, 59);
border-top-left-radius:5px; 
border-bottom-left-radius:5px; 
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string> Confirm Password </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color:rgb(59, 59, 59);
border-top-left-radius:5px; 
border-bottom-left-radius:5px; 
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string> Email  </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color:rgb(59, 59, 59);
border-top-left-radius:5px; 
border-bottom-left-radius:5px; 
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string> Gender </string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(50, 50, 50);
font: 12pt &quot;MS Shell Dlg 2&quot;;
border-top-right-radius:5px;
color:rgb(255, 255, 255);
</string>
   </property>
   <item>
    <property name="text">
     <string>      Male</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>     Female</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(50, 50, 50);
font: 11pt &quot;MS Shell Dlg 2&quot;;
border-top-right-radius:5px; 
border-bottom-right-radius:5px; 
color:rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>is empty</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(50, 50, 50);
font: 11pt &quot;MS Shell Dlg 2&quot;;
border-top-right-radius:5px; 
border-bottom-right-radius:5px; 
color:rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>is empty</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(50, 50, 50);
font: 11pt &quot;MS Shell Dlg 2&quot;;
border-top-right-radius:5px; 
border-bottom-right-radius:5px; 
color:rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>is empty</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(50, 50, 50);
font: 11pt &quot;MS Shell Dlg 2&quot;;
border-top-right-radius:5px; 
border-bottom-right-radius:5px; 
color:rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>is empty</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(50, 50, 50);
font: 11pt &quot;MS Shell Dlg 2&quot;;
border-top-right-radius:5px; 
border-bottom-right-radius:5px; 
color:rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>is empty</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(50, 50, 50);
font: 12pt &quot;MS Shell Dlg 2&quot;;
border-radius:5px; 
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(50, 50, 50);
font: 12pt &quot;MS Shell Dlg 2&quot;;
border-radius:5px; 
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="game_recources.qrc">:/game_backgrounds/pics_project/sign.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>701</width>
     <height>461</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="game_recources.qrc">:/game_backgrounds/pics_project/sign up.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_8</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>comboBox</zorder>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>lineEdit_4</zorder>
  <zorder>lineEdit_5</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
 </widget>
 <resources>
  <include location="game_recources.qrc"/>
 </resources>
 <connections/>
</ui>
